packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -571,7 +571,7 @@ export class CallContext {
   public setDeviceManagerCameras(cameras: VideoDeviceInfo[]): void {
     this.setState(
       produce(this._state, (draft: CallClientState) => {
-        if (!!draft.deviceManager.cameras && cameras.length > 0) {
+        if ((!draft.deviceManager.cameras || draft.deviceManager.cameras.length === 0) && cameras.length > 0) {
```

Comment:
The logic is supposed to be 'only if we previously had no cameras and getting new cameras then we should select the first one by default'